### ISSUE TITLE
Make sure images used for demo and examples have tag

### ISSUE THREAD

**Issue by dchen1107** [kubernetes/kubernetes#3698]
We recently updated some images we are using to have proper tag / version, such as cadvisor, fluentd-elasticsearch etc. But there are still having a lot of images without tag.

```
find examples/ -type f | xargs grep "image: "
examples/hazelcast/hazelcast-controller.yaml:            image: pires/hazelcast-k8s
examples/hazelcast/README.md:        image: pires/hazelcast-k8s
examples/hazelcast/README.md:            image: pires/hazelcast-k8s
examples/hazelcast/hazelcast.yaml:        image: pires/hazelcast-k8s
examples/walkthrough/README.md:        image: dockerfile/nginx
examples/walkthrough/README.md:        image: dockerfile/redis
examples/walkthrough/README.md:        image: dockerfile/nginx
examples/walkthrough/README.md:        image: kubernetes/git-monitor
examples/walkthrough/pod1.yaml:        image: dockerfile/nginx
examples/walkthrough/pod-with-http-healthcheck.yaml:        image: dockerfile/nginx
examples/walkthrough/pod2.yaml:        image: dockerfile/redis
examples/walkthrough/replication-controller.yaml:            image: dockerfile/nginx
examples/walkthrough/k8s201.md:            image: dockerfile/nginx
examples/walkthrough/k8s201.md:        image: dockerfile/nginx
examples/liveness/exec-liveness.yaml:      - image: busybox
examples/liveness/http-liveness.yaml:      - image: kubernetes/liveness
examples/gce-pd/testpd.yaml:      - image: kubernetes/pause

```

**Comment by pires** [comment]
`hazelcast` example Docker image is now tagged on `quay.io/pires/hazelcast-kubernetes` but @brendandburns and I agree it should be taken over by this project maintainers.

`quay.io/pires/hazelcast-kubernetes:0.3` is waiting on [this issue latest comments](https://github.com/GoogleCloudPlatform/kubernetes/issues/7444#issuecomment-106080138) to become clear and tested accordingly to be included in the doc.

**Comment by kargakis** [comment]
Related: https://github.com/kubernetes/kubernetes/issues/1827

**Comment by kargakis** [comment]
@kubernetes/sig-apps-misc

**Comment by ahmetb** [comment]
We moved the examples to their own repo (https://github.com/kubernetes/examples) for further maintenance.  All the maintained examples now either use Docker Hub or gcr.io/google-samples, and they all use tags for images.

Looks like this issue has been open with little to no recent activity, if you think it's still relevant, please create a new issue in the new repo.
